app/person.py

```diff
@@ -0,0 +1,24 @@
+class Person:
+	"""An overal class for person"""
+
+	def __init__(self, person_name):
+
```

Comment:
You can rename person_name to first_name and have `def __init__(self, first_name, last_name):`